package.json

```diff
@@ -8,10 +8,14 @@
   },
   "devDependencies": {
     "@types/node": "^12.12.21",
+    "@yarnpkg/sdks": "^3.0.0-rc.43",
```

Comment:
what does this package do?

Reply:
SDKS updates what is in the sdks folder: prettier, typescript patches, etc. Making it a dependency means you can use `yarn sdks` instead of `yarn dlx @yarnpkg/sdks`